packages/sdk/src/types/deploy/deploy-metadata.ts

```diff
@@ -172,7 +172,7 @@ export interface VoteContractDeployMetadata {
   /**
    * The minimum amount in governance token owned to be able to create a proposal
    */
-  proposal_token_threshold?: BigNumberish;
+  proposal_token_threshold?: Exclude<BigNumberish, Bytes>;
```

Comment:
this looks off? what was the issue here

Reply:
our `BigNumberSchema` does not include the ethers type `Bytes`, we can either include it in our BN schema or we have to remove it here (otherwise the types don't match)